requirements.txt

```diff
@@ -40,3 +40,4 @@ vobject==0.6.6
 wsgiref==0.1.2
 XlsxWriter==0.7.7
 xlwt==0.7.5
+openupgradelib>=1.1.0
```

Comment:
Nice! I was going through the Travis logs for a hint on this but I did not find anything. But you are right https://pypi.python.org/pypi/openupgradelib/1.2.0